docs/Gateways/index.md

```diff
@@ -42,3 +42,29 @@ You need to add the domain and in similar fashion to how its added to a site in
 Once the verification is completed you will see the status of the gateway change to `Active` and you will be able to use it to serve your content. You can access the content on your gateway via the domain like this `my-gateway-url/ipfs/<cid>` and `my-gateway-url/ipns/<cid>`
 
 You can set up more than one domain per gateway. To do this you can click on the three dots icon in the gateway row and select the `Add Domain` option. This will open a modal where you can add a new domain to the gateway.
+
+### Priamry Domains
+
+![](../images/primary-domain-gw.png)
+
+Whe you have correctly seted up your first domain it will be marked with the tag of `Primary`. The `Primary` domain is the main domain for a site and it will be used inside of the platform for every action that is related to files as source of truth. For example, when you store a file, the domain that will be used in the `view file` button will be the `Primary` domain.
```

Comment:
Mb we want to explain that the primary domain for pgw is across all pgw in a same project? I didn't add it, just changed site for private gateways

```suggestion
Whe you have correctly set up your first domain it will be marked with the tag of `Primary`. The `Primary` domain is the main domain for private gateways and it will be used inside of the platform for every action that is related to files as source of truth. For example, when you store a file, the domain that will be used in the `view file` button will be the `Primary` domain.
```